packages/lib/src/helpers/ast-transformer.ts

```diff
@@ -99,9 +99,6 @@ const convertCriteria = (criteria: AstTopLayer): QueryItem | undefined => {
         system: checkedValues[0].system,
         values: convertedValues,
     };
-    if (catalogueCategory != undefined && "description" in catalogueCategory) {
-        convertedCriteria.description = catalogueCategory.description;
-    }
```

Comment:
Note to reviewer: removed this because Category never has the description property

Reply:
Good point, I think this is a really old data element. I think this was something project specific. Can be removed.